shentu/chain.json

```diff
@@ -35,47 +35,38 @@
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/ShentuChain/mainnet/main/shentu-2.2/genesis.json"
     },
-    "recommended_version": "v2.7.1",
+    "recommended_version": "v2.8.0",
     "compatible_versions": [
-      "v2.7.0",
-      "v2.7.1"
+      "v2.8.0"
     ],
     "ibc_go_version": "v4.2.1",
     "ics_enabled": [
       "ics20-1"
     ],
     "binaries": {
-      "linux/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.7.1/shentud_2.7.1_linux_amd64",
-      "darwin/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.7.1/shentud_2.7.1_arm64_macos",
-      "windows/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.7.1/shentud_2.7.1_win_x86_64.exe"
+      "linux/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_linux_amd64",
+      "darwin/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_arm64_macos",
+      "windows/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_win_x86_64.exe"
     },
     "versions": [
       {
         "name": "v2.6.0",
         "height": 10485430,
         "proposal": 14,
         "tag": "v2.6.0",
-        "recommended_version": "v2.6.0",
```

Comment:
Please re-add the recommended and compatible versions back in (for all versions it was removed)

Reply:
added